pages/common/tailscale.md

```diff
@@ -32,6 +32,6 @@
 
 `tailscale bugreport`
 
-- Display help for a subcommand:
+- Display debug helper commands (undocumented):
 
-`tailscale {{subcommand}} --help`
+`tailscale debug --help`
```

Comment:
Not sure, why not create the `tailscale-debug` page?

Reply:
Its an unstable subcommand. So i thought just to give a heads up to the reader; when the user lands in the main `tailscale` tldr-page.
This may prove as a good starting point to debug.
Personally, I was looking for something like this and given `tailscale --help` does not show its existence, its unlikely anyone would know what they are looking for.
My rationale, nudge the user about its existence - Who knows it might turn out to be useful :)

Reply:
After looking into this, a) yes we can add this but not under the current PR. b) since this is open for close to a year, is it really relevant?

Reply:
> After looking into this, a) yes we can add this but not under the current PR. b) since this is open for close to a year, is it really relevant?

Agreed, we can add a dedicated page for this Experimental command if required, but the `debug` command is still unstable. Will close this PR for now, feel free to create a new one for the separate page.